xcode/Subconscious/Shared/Components/Deck/CardStackView.swift

```diff
@@ -34,37 +83,31 @@ struct CardView: View {
         VStack(alignment: .leading, spacing: AppTheme.unit2) {
             switch entry.card {
             case let .entry(entry, _, backlinks):
-                SubtextView(
-                    peer: entry.toPeer(),
-                    subtext: entry.excerpt,
+                CardContentView(
+                    entry: entry,
+                    backlinks: backlinks,
                     onLink: onLink
                 )
-                // Opacity allows blendMode to show through
-                .foregroundStyle(.primary.opacity(0.8))
-                .accentColor(highlight)
-                
-                Spacer()
-                
-                HStack {
-                    // Consider: should the date go here?
-                    Text(
-                        entry.address.markup
-                    )
-                    .lineLimit(1)
-                    
-                    if !backlinks.isEmpty {
-                        Spacer()
+            case let .prompt(message, entry, _, backlinks):
+                VStack(alignment: .leading, spacing: 0) {
+                    HStack {
+                        Image(systemName: "sparkles")
                         
-                        HStack {
-                            Image(systemName: "link")
-                            Text("\(backlinks.count)")
-                        }
+                        Text(message)
                     }
+                    .padding(DeckTheme.cardPadding)
+                    .foregroundStyle(highlight)
+                    .font(.subheadline)
+                    
+                    Divider()
+                        .overlay(highlight)
```

Comment:
Nit: this divider looks heavy to my eye. I think it should visually read as a crease in the paper.